session-replay/Sources/DatadogSessionReplay/Recorder/SnapshotBuilder/NodeRecorders/UILabelRecorder.swift

```diff
@@ -39,6 +40,8 @@ internal struct UILabelWireframesBuilder: NodeWireframesBuilder {
     let textColor: CGColor?
     /// The font used by the label.
     let font: UIFont?
+    /// Text obfuscator for masking text (`nil` if text should not be obfuscated for this element).
+    let textObfuscator: TextObfuscator?
```

Comment:
We could have a `TextObfuscator` protocol with a `NOPTextObfuscator` and make this non-optional, WDYT?
Not a change request, it's totally fine with struct+optional.

Reply:
Good idea 👍, it's less optionals in downstream elements (wireframe builder), so I like it. Done 👌.